spec/features/create_a_user_spec.rb

```diff
@@ -1,15 +1,15 @@
-require 'rails_helper'
-
-RSpec.feature 'Create a User', :type => :feature do
-  scenario 'New users can sign up' do
-      visit '/'
-      click_link 'Sign Up'
-      expect(page).to have_content 'Sign Up Here'
+# frozen_string_literal: true
 
-      fill_in 'Name', with: 'MyFirstName'
-      fill_in 'Email', with: 'MyEmailAddress'
-      click_button 'Sign Up'
+require 'rails_helper'
 
-      expect(page).to have_text('You have successfully signed up!')
+RSpec.feature 'Create a User', type: :feature do
+  scenario 'New users can sign up with valid name and email' do
+    user = User.new(name: 'Olivia', email: 'olivia@aivilo.com')
+    visit '/'
+    click_link 'Sign Up'
+    fill_in 'Name', with: user.name
+    fill_in 'Email', with: user.email
+    click_button 'Sign Up'
+    assert user.save
```

Comment:
Not clear why you moved from an [Rspec expectation](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers) to a TestUnit assertion. I'd recommend using the `expect` syntax.